app/controllers/staff/activities_controller.rb

```diff
@@ -5,7 +5,8 @@ class Staff::ActivitiesController < Staff::BaseController
 
   def index
     @organisation_id = organisation_id
-    @activities = policy_scope(Activity.where(organisation: organisation_id))
+    @activities = policy_scope(Activity.where(organisation: organisation_id).where.not(programme_status: ["completed", "stopped", "cancelled"])
+      .or(Activity.where(organisation: organisation_id, programme_status: nil)))
```

Comment:
This feels like something it would be worth refactoring into a scope on `Activity`, to make the intent clearer, for example:

```rb
class Activity
  scope :historic, -> {
    where.not(programme_status: ["completed", "stopped", "cancelled"]).or(where(programme_status: nil))
  }
end
```

We could then write in the controller:

```rb
Activity.historic.where(organisation: organisation_id)
```

Reply:
Nice idea! I like how your example reads. I'll change my code. Thanks for the suggestion!